modules/nw-route-set-or-delete-http-headers.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * networking/route-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-route-set-or-delete-http-headers_{context}"]
+= Setting or deleting HTTP request and response headers in a route
+
+Certain HTTP request and response headers might need to be set or deleted for compliance purposes or other reasons. You can set or delete these headers either for all routes served by an Ingress Controller or for specific routes.
+
+For example, you might want to enable a web application to serve content in alternate locations for specific routes if that content is written in multiple languages, even if there is a default global location specified by the Ingress Controller serving the routes.
+
+The following procedure creates a route that sets the Content-Location HTTP request header so that the URL associated with the application, `\https://app.example.com`, directs to the location `\https://app.example.com/lang/en-us` that way anyone using that specific route is accessing web content written in American English.
```

Comment:
The part of starting with "that way..." seems like it's turning into a run-on sentence at this point. I'd recommend breaking this up into two sentences.